Octokit.Tests.Integration/Clients/ReleasesClientTests.cs

```diff
@@ -645,7 +645,7 @@ public TheUploadAssetMethod()
             {
                 PageSize = 1,
                 PageCount = 1,
-                StartPage = 2
+                StartPage = 1
```

Comment:
hmmm confusing, it says this is an outdated diff but i cant see where it was changed back :confused: